assets/css/filtersection.css

```diff
@@ -55,7 +38,353 @@
     background-color: var(--verylightgray);
 }
 
-/* Show the dropdown menu (use JS to add this class to the .dropdown-content container when the user clicks on the dropdown button) */
-.show {
+#filters .filter-links {
```

Comment:
In general, it's best to avoid styling with ids (e.g.`#filters`), because ids refer to one specific element and don't encourage reusable styles. Instead, I would save ids for accessing specific, one-off, elements from JavaScript. Nowadays, there's not much use for id.

Reply:
Further, I encourage you to simplify your selectors to just one class unless absolutely necessary. Otherwise, you will run into specificity issues.

Is there any reason you can't just do `.filter-links`?